cves/CVE-2017-14170.yml

```diff
@@ -167,10 +193,11 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Found by Xiaohei and Wangchu of the Alibaba Security Team.
```

Comment:
My comment in the other file at this section applies here too. (Except for the part where these guys discovered one of my vulnerabilities - Alibaba did not.)

Reply:
As I said earlier, I just misunderstood the instructions. I've updated it to reflect my research